dvc/repo/__init__.py

```diff
@@ -489,15 +489,6 @@ def dvcignore(self):
     def close(self):
         self.scm.close()
 
-    @staticmethod
```

Comment:
Not sure why you had to remove this one. You can pass a tmp path to it.

Reply:
Plus this is breaking an API :)

Reply:
This has nothing to do with API. Should never been there form the start.

Reply:
@Suor Well, but it was. And it presents a useful helper, similar to `Repo.clone` from GitPython.

Reply:
This was simply a wrapper around `Git.clone()` not used by anyone, not corresponding to any cli thing, not adding any own logic. Just a proxy method.

Reply:
And I can't use it because it's creating a `Repo` object, which I don't want. I want git clone, so I use git clone.

Reply:
Probably not. You were closing and reopening `Repo()` in prev implementation. Anyway, why should I want it?

Reply:
Even if I wanted that the proper way would be making:
```python
Git.clone(...)
repo = Repo(path)
```
This two actions are optionally consequential in this single scenario, this is a bad reason to create a method.